Dockerfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
Maybe rename it to something like `hepcrawl_base.dockerfile`, so it's clear what it is, this will require changing the 'dockerfile' option in the docker-compose file to point to the new path.